docs/index.html

```diff
@@ -185,7 +185,7 @@ <h2 class="section__title">Design</h2>
           Sketch UI Kit bundle includes ready-to-use, state-of-the-art drafts of SAP layouts, patterns and components in Sketch. You are welcome to use the UI kit to visualize your SAP app. They are easy to use and give you a realistic impression of your final design.
         </p>
         <div class="section__container-button">
-          <a role="button" class="section__button" href="./images/ui-kit/Fundamental_Library_UI_Kit_V.0.9.0.zip">Download</a>
+          <a role="button" class="section__button" href="./images/ui-kit/Fundamental_Library_UI_Kit_V.0.10.0.zip">Download</a>
         </div>
         <h3 class="section__subtitle">What's in UI Kit bundle?</h3>
```

Comment:
**Quick edit**
What's in the UI Kit bundle?

Reply:
font 72, icons, and ui kit